docs/MapColonies/vector/Services/geocoding/README.md

```diff
@@ -432,6 +432,51 @@ Almost all of our routes consists of the same common query parameters: `geo_cont
 | limit | Number | 5 | [Click here](#item-search)  | By default, we will return our top 5 features that match the query. You can change the limit and set it from 1 to 15 maximum results. If there are few results, the response may contain less than limit, but the importance is limiting the maximum returned values. |
 | disable_fuzziness | Boolean  | false | [Click here](#exact-tile-search) | Fuzziness is on by default. If you want exact match, you may set `disable_fuzziness: true`. |
 
+## Generic Response Object
+
+As part of Geocoding's architecture, we created a generic response object. All `/search` routes will return the same expected response object and properties. <br/>
+📝 **_Note:_** Responses in the different routes may have more properties than described in the generic response object.
+
+```json
```

Comment:
Don't you want to explain each field in more detail outside of the response example?

Reply:
No.